src/current/v24.2/install-cockroachdb-windows.md

```diff
@@ -24,20 +24,29 @@ Use one of the options below to install CockroachDB. To upgrade an existing clus
 <section id="download-the-binary-windows" markdown="1" class="install-option">
 <h2 id="install-binary">Download the executable</h2>
 
-1. Using PowerShell, run the following script to download the [CockroachDB {{ page.release_info.version }} archive for Windows](https://binaries.cockroachdb.com/cockroach-{{ page.release_info.version }}.windows-6.2-amd64.zip) and copy the binary into your `PATH`:
+You can download and install CockroachDB for Windows in two ways. Either:
 
-    {% include_cached copy-clipboard.html %}
-    ~~~ powershell
-    $ErrorActionPreference = "Stop"; [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12;$ProgressPreference = 'SilentlyContinue'; $null = New-Item -Type Directory -Force $env:appdata/cockroach; Invoke-WebRequest -Uri https://binaries.cockroachdb.com/cockroach-{{ page.release_info.version }}.windows-6.2-amd64.zip -OutFile cockroach.zip; Expand-Archive -Force -Path cockroach.zip; Copy-Item -Force "cockroach/cockroach-{{ page.release_info.version }}.windows-6.2-amd64/cockroach.exe" -Destination $env:appdata/cockroach; $Env:PATH += ";$env:appdata/cockroach"
-    ~~~
+- (Recommended) Visit [Releases]({% link releases/index.md %}) to download the CockroachDB {{ page.version.version }} archive for Windows. The archive contains the `cockroach.exe` binary. Extract the archive and optionally copy the `cockroach.exe` binary into your `PATH` so you can execute [cockroach commands]({% link {{ page.version.version }}/cockroach-commands.md %}) from any shell.
 
-    {{site.data.alerts.callout_success}}
-    To run a PowerShell script from a file, use syntax like `powershell.exe -Command "{path_to_script}"`.
-    {{site.data.alerts.end}}
+- Instead of downloading the binary directly, you can follow these steps to use PowerShell to download and install CockroachDB.
+
+1. Visit [Releases]({% link releases/index.md %}) and copy the link to the full CockroachDB binary you want to install.
+1. Save the following PowerShell script and replace the following:
+    - `{ DOWNLOAD_URL }`: the download URL.
```

Comment:
I am trying to get away from us having to maintain logic that shows the download URLs like this in favor of sending them to the official docs. If we are going to do what you suggest, we might as well just keep the `curl` commands around. This will always break when there is a Cloud-first phase.

Reply:
I thought you might be doing that, but my main point is that the powershell snippet already contains a separate path that seems to be broken, unless I'm mistaken. 

```
cockroach/cockroach-{{ page.version.version }}.windows-6.2-amd64/cockroach.exe
```